### PR TITLE
Implemented the Review model

### PR DESCRIPTION
**Pull request by AhmadAlBarasy**

### CODE REVIEW

File: src/models/review.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    rating: {
+        type: DataTypes.FLOAT(2, 1),
+        validate: {
+            min: 1.0,
+            max: 5.0
+        }
+    },
+    comment: {
+        type: DataTypes.STRING(500)
```
Comment (MahmoudMDarabaih) [inline src/models/review.ts:20]: Shouldn't we add allow null false?

**Review by MahmoudMDarabaih — CHANGES_REQUESTED** [review]
added some comments, please check.

File: src/models/review.ts (hunk 1 above)
Comment (AhmadAlBarasy) [reply src/models/review.ts:20]: I think a user should be able to add a review without giving a comment, just a rating value and he's good to go, what do you think?

Comment (MahmoudMDarabaih) [reply src/models/review.ts:20]: ok, good to go, ill approve.

**Review by MahmoudMDarabaih — APPROVED** [review]

**Pull request merged**